src/brogue/Items.c

```diff
@@ -3387,7 +3387,10 @@ short getLineCoordinates(short listOfCoordinates[][2], const short originLoc[2],
             boolean isMonster = monst
                 && !(monst->bookkeepingFlags & MB_SUBMERGED)
                 && !monsterIsHidden(monst, monsterAtLoc(originLoc[0], originLoc[1]));
-            boolean isAllyOfCaster = ((monst && monst->creatureState == MONSTER_ALLY) == isCastByPlayer);
+            creature *caster = monsterAtLoc(originLoc[0], originLoc[1]);
+            boolean isAllyOfCaster = // true if monster and caster are on the same side
+                ((monst && monst->creatureState == MONSTER_ALLY) ==
+                (caster && caster->creatureState == MONSTER_ALLY));
```

Comment:
Might we use `monstersAreTeammates`?

Reply:
Certainly; changed.

Reply:
Ah no, I mean the function `monstersAreTeammates`!

Reply:
Oops! I didn't know that function existed 🤣I'll use it for when the caster is a monster.

Reply:
I don't think you need to restrict it, it works for player and allies too

Reply:
Sorry, I meant "creature", as opposed to turrets... Actually, when the caster is a turret, I can use it too! See 
4957ba4.